### ISSUE TITLE
Support ahash (the latest version v0.8.11)

### ISSUE THREAD

**Issue by MITSUBOSHI** [metrics-rs/metrics#464]
Hi, I'm trying to upgrade crate dependencies in my project.
But the dependency conflict like below occurs. 
 
I would like the maintainers to support ahash v0.8.11.

Thank you in advance.

### Dependency Conflict (by renovate)
```
Command failed: cargo update --config net.git-fetch-with-cli=true --manifest-path my_app/Cargo.toml --workspace
    Updating crates.io index
error: failed to select a version for `ahash`.
    ... required by package `metrics v0.22.1`
    ... which satisfies dependency `metrics = "^0.22.1"` of package `my_app v0.1.0 (/workdir/my_app)`
versions that meet the requirements `>=0.8, <=0.8.7` are: 0.8.6, 0.8.7, 0.8.5, 0.8.4

all possible versions conflict with previously selected packages.

  previously selected package `ahash v0.8.11`
    ... which satisfies dependency `ahash = "^0.8.11"` of package `aws-sdk-s3 v1.19.0`
    ... which satisfies dependency `aws-sdk-s3 = "^1.19.0"` of package `my_app v0.1.0 (/workdir/my_app)`

failed to select a version for `ahash` which could resolve this conflict
```

- [aws_sdk_s3](https://crates.io/crates/aws-sdk-s3): v1.19.0
  - It has a dependency on ahash (version = '0.8.11')
  - https://github.com/awslabs/aws-sdk-rust/blob/f8493097187e6b7c35b29f182ea897dd61b0a4f6/sdk/s3/Cargo.toml#L16-L17

**Comment by tobz** [comment]
This should be resolved once we cut the next release of `metrics`, which has an update to drop the range constraint on `ahash` and switch to `^0.8.8`.

**Comment by tobz** [comment]
Released in `metrics@v0.22.2`.

**Comment by MITSUBOSHI** [comment]
Thank you!! The issue has been resolved by new release.
